src/stylesheet/index.js

```diff
@@ -45,6 +45,7 @@ const resolve = (styles, container) => {
 
   styles = flatten(styles);
   styles = resolveMediaQueries(styles, container);
+  // something here to resolve @font-face styles?
```

Comment:
I would leave `@font-face` css support for a separate issue, and stick with the `Font.register` API for the moment